backend/prompt_studio/prompt_studio_registry/prompt_studio_registry_helper.py

```diff
@@ -176,7 +176,6 @@ def update_or_create_psr_tool(
             obj, created = PromptStudioRegistry.objects.update_or_create(
                 custom_tool=custom_tool,
                 created_by=custom_tool.created_by,
```

Comment:
@johnyrahul Will created_by field cause an IntegrityError ?

Reply:
no @harini-venkataraman . The field will always be fixed for custom tools.